design/common.blocks/checkbox/_theme/checkbox_theme_islands.post.css

```diff
@@ -1,5 +1,5 @@
 :block(checkbox):mod(theme islands) {
```

Comment:
`rebem css` is quite stupid but such magic is possible in `pobem`.
I believe we should have generic implementation in libs and provide users to write the way they want.